From d75874557dad0059b9399e7350f2d4ee6606dfab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 12 Jan 2015 12:14:59 -0500
Subject: [PATCH] core/mount: remove "fail" again

deb6120920 'man: there's actually no "fail" fstab option, but only
"nofail" removed it from our documentation, which I missed.
fstab(5) only mentions "auto", "noauto", and "nofail". Stick to
those three.

(cherry picked from commit 76c37ab6d205c95e8f43ffbc19553b936330195a)
---
 src/core/mount.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/mount.c b/src/core/mount.c
index 5414572a66..8afa2e6dda 100644
--- a/src/core/mount.c
+++ b/src/core/mount.c
@@ -901,7 +901,7 @@ static void mount_enter_mounting(Mount *m) {
                 _cleanup_free_ char *opts = NULL;
 
                 r = fstab_filter_options(m->parameters_fragment.options,
-                                         "nofail\0" "fail\0" "noauto\0" "auto\0", NULL, NULL, &opts);
+                                         "nofail\0" "noauto\0" "auto\0", NULL, NULL, &opts);
                 if (r < 0)
                         goto fail;
 
